doc/source/whatsnew/v0.18.2.txt

```diff
@@ -317,6 +317,7 @@ Bug Fixes
 - Bug in ``groupby`` where ``apply`` returns different result depending on whether first result is ``None`` or not (:issue:`12824`)
 
 
+- Bug in ``pd.to_numeric`` when ``errors='coerce'`` returns same dataframe/series when dataframe/series contains non-hashable objects (:issue:`13324`)
```

Comment:
Bug in ``pd.to_numeric`` when ``errors='coerce'`` and input contains non-hashable objects....